### PR TITLE
SS-1807 Docs feedback element

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

**Comment by KaustubhKumar05** [comment]
Metabase integration for analyzing text messages is pending

File: components/Feedback.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    React.useEffect(() => {
+        if (inputRef.current) inputRef.current.focus();
+    }, [clickedEmoji]);
+
+    return (
+        // @ts-ignore
+        <Box ref={feedBackRef} css={{ maxWidth: '200px' }}>
+            <Text
+                variant="tiny"
+                css={{ fontWeight: 'bold', color: '$textHighEmp', textTransform: 'uppercase' }}>
```
Comment (arjunattam) [inline components/Feedback.tsx:28]: these styles are slightly different from the styles on `on this page`. ones i could find:

* font weight should be 500 (bold gives 700)
* letter spacing needs to be higher

File: components/Toc.tsx [hunk 2]
```diff
@@ -46,7 +47,8 @@ const TocContainer = ({ activeHeading, activeSubHeading, CurrentDocsSlug }) => {
                     </div>
                 </>
             ) : null}
-
+            <hr />
```
Comment (arjunattam) [inline components/Toc.tsx:50]: for server-side API pages, the "run in postman" button is not vertically centered

Screenshot 2022-12-14 at 12 06 40 AM

**Review by arjunattam — APPROVED** [review]

File: components/Feedback.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+import { currentUser } from '../lib/currentUser';
+
+const emojis = [{ score: 1 }, { score: 2 }, { score: 3 }, { score: 4 }];
+
+const Feedback = () => {
+    const [showTextBox, setShowTextBox] = React.useState(false);
+    const [clickedEmoji, setClickedEmoji] = React.useState(0);
+    const [submitSuccessful, setSubmitSuccessful] = React.useState(false);
+    const [message, setMessage] = React.useState('');
+    const feedBackRef = React.useRef<HTMLDivElement | null>();
```
Comment (hardik1003) [inline components/Feedback.tsx:13]: ?

**Review by hardik1003 — APPROVED** [review]

File: components/Feedback.tsx (hunk 3 above)
Comment (KaustubhKumar05) [reply components/Feedback.tsx]: Can you be more descriptive?

**Review by hardik1003 — APPROVED** [review]

**Pull request merged**